docs/installation/database/passing-configuration.rst

```diff
@@ -0,0 +1,430 @@
+
+.. py:currentmodule:: datacube
+
+ODC Configuration (Details)
+***************************
+
+Open Data Cube configuration files, supported configuration options, and basic use cases and defaults
+are described in :doc:`configuration`.
+
+The Open Data Cube uses configuration files and/or environment variables to determine how to connect to databases.
+
+Overview
+========
+
+The normal flow of configuration processing can be bypassed by `explicitly passing`_ in a complete configuration.
+Such explicit configuration blocks reading of configuration from environment variables.
+
+Otherwise, configuration is read from a `configuration file`_ and/or `environment variables`_.
+
+One configuration can define multiple environments, so users must `choose one`_.
+
+The configuration engine in 1.9 is not 100% compatible with the previous configuration engine.  Advanced
+users and developers upgrading 1.8 systems should read the `migration notes`_.
+
+.. _`explicitly passing`: #Explicit-configurations
+.. _`configuration file`: #File-configurations
+.. _`choose one`: #The-Active-Environment
+.. _`environment variables`: #Generic-Environment-Variable-Overrides
+.. _`migration notes`: #Migrating-from-datacube-1.8
+
+1. Explicit configuration
```

Comment:
This heading is "explicit configuration", but the following subsections appear to describe "raw configuration", and then the `ODC_CONFIG` environment variable comes in and doesn't look raw or explicit at all.

Is it too late to rename `ODC_CONFIG` to `ODC_RAW_CONFIG` so it is consistent with the other naming? (Or other way around, rename the `raw_config` parameters to just `config`?)